docs/en/observability/uptime-set-up.asciidoc

```diff
@@ -0,0 +1,94 @@
+[[uptime-set-up]]
+= Set up monitors
+
+++++
+<titleabbrev>Set up monitors</titleabbrev>
+++++
+
+When setting up an uptime monitor, you will use a provided Docker image to run the monitor and send data back to Elasticsearch. 
+There are two approaches to configuring the monitor and connecting it to Elasticsearch.
```

Comment:
I think here we must mention what these two approaches are and be explicit about which one we recommend.